pkg/worker/handler/stack/stack_test.go

```diff
@@ -0,0 +1,68 @@
+package stack
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_Worker_Handler_Stack_rooSta(t *testing.T) {
+	testCases := []struct {
+		arn string
+		roo bool
+	}{
+		// Case 000
+		{
+			arn: "server-test-SpectaStack-XQCAJRTZL8RF",
+			roo: false,
+		},
```

Comment:
This is more or less the same unit test that we had before. And funny enough, we can reuse it for the updated approach. So the test remains basically the same, we just adjusted it for the refactored version.